lib/livebook/teams/deployment_group.ex

```diff
@@ -36,6 +36,6 @@ defmodule Livebook.Teams.DeploymentGroup do
   def changeset(deployment_group, attrs \\ %{}) do
     deployment_group
     |> cast(attrs, [:id, :name, :mode, :hub_id, :clustering, :zta_provider, :zta_key])
-    |> validate_required([:name, :mode])
+    |> validate_required([:name])
```

Comment:
We want it for creation, and for edit it should pass because the deployment group we edit always has it set, so it's fine to keep?

Reply:
Not if we take it out of the form like now

Reply:
If mode is not passed but `deployment_group.mode` is present, this validation shouldn't fail. Does it fail?

Reply:
It's failing now because we are not passing the `deployment_group.mode` anywhere

Reply:
Insert it as a hidden field then?

Reply:
To be clear, insert the mode as a hidden field.